src/document/operation/changeoperation.js

```diff
@@ -77,45 +80,15 @@ CKEDITOR.define( [ 'document/operation/operation', 'ckeditorerror' ], ( Operatio
 			}
 
 			// Remove or change.
-			if ( oldAttr !== null ) {
+			if ( oldAttr !== null && newAttr === null ) {
 				for ( value of this.range ) {
-					if ( !value.node.hasAttr( oldAttr ) ) {
-						/**
-						 * The attribute which should be removed does not exists for the given node.
-						 *
-						 * @error operation-change-no-attr-to-remove
-						 * @param {document.Node} node
-						 * @param {document.Attribute} attr
-						 */
-						throw new CKEditorError(
-							'operation-change-no-attr-to-remove: The attribute which should be removed does not exists for given node.',
```

Comment:
Yes it was connected with some problems when I was doing ChangeOperation by ChangeOperation OT. I don't remember the case exactly but in oldAttr was wrong in that scenario and it required making more complicated OT script. Maybe this already got solved during one of a few refactorings of all those scripts. I'll check it once again.